src/solver/conv_multipass_wino3x3WrW.cpp

```diff
@@ -364,114 +363,118 @@ bool ConvWinograd3x3MultipassWrW<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>
 // HIP backend required for sending ptr (buffer + offset)
 // ROCBLAS for GEMM step
 
-#if(MIOPEN_BACKEND_HIP && MIOPEN_USE_ROCBLAS)
-    static const int wino_data_tile   = std::max(WinoDataH, WinoDataW);
-    static const int wino_filter_tile = std::max(WinoFilterH, WinoFilterW);
-
-    if(wino_data_tile == 3 && wino_filter_tile == 2)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X2{}) ||
-           params.kernel_stride_h == 1)
-            return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 3)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X3{}) ||
-           params.kernel_stride_h == 1)
-            return false;
-
-    const std::string name = params.GetStream().GetDeviceName();
-#if WORKAROUND_SWDEV_234193
-    if(params.IsFp16() && (StartsWith(name, "gfx908") || StartsWith(name, "gfx906")))
+#if(MIOPEN_BACKEND_HIP && (MIOPEN_USE_ROCBLAS || MIOPEN_USE_MIOPENTENSILE))
+#if(!MIOPEN_USE_ROCBLAS)
```

Comment:
This (and next change) looks like unnecessary complication. Simply insert this:
```C++
/// miopentensile currently supports only FP32.
/// \todo Remove this when FP16/BF16 support on miopentensile path is ready.
#if !MIOPEN_USE_ROCBLAS
    if(!params.IsFp32())
        return false;
```

Reply:
will do